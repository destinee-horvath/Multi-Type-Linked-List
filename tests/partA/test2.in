NEW 5
#
$
@@@
(
)
NEW 4
long
string
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
end
VIEW ALL
VIEW 0
TYPE 0
VIEW 1
TYPE 1
