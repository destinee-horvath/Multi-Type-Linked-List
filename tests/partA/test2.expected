List 0: # -> $ -> @@@ -> ( -> )
List 1: long -> string -> aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa -> end
Number of lists: 2
List 0
List 1
# -> $ -> @@@ -> ( -> )
char -> char -> string -> char -> char
long -> string -> aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa -> end
string -> string -> string -> string
